governance.md

```diff
@@ -0,0 +1,224 @@
+Refer to the section on [Community Groups](#community-groups) and [SIGs](#sigs) in particular for details on how this is performed.
+5. **Truly Open Community** : The best way to build a sustainable open source community is to ensure the community is a cohesive, inclusive, and equitable place where all the voices are heard and anyone can rise to leadership positions.
+
+### Code of Conduct
+
+We are using the [upstream OpenInfra Foundation Code of Conduct](https://openinfra.dev/legal/code-of-conduct)(CoC).
+In creating the next Governance 1.1 we will have an actual process to make decisions as a community. We can then either formally ratify this same CoC, or the Community SIG can convene a cross-project working group to discuss and decide an inclusive and equitable process for a new code of conduct.
+
+### Community membership
+> :notebook: **Note** This document is a work in progress, especially these membership guidelines
+
+We are opening the Governance 1.0 directly borrowing from the [Kubernetes community group definitions](https://github.com/kubernetes/community/blob/master/community-membership.md), then reviewing and refining for the Governance 1.1.
+
+This section outlines the various responsibilities of contributor roles in Op1st.
+The Op1st project is subdivided into subprojects under SIGs.
+Responsibilities for most roles are scoped to these subprojects.
+
+A contribution can take many, many forms, such as: documentation, code, best-practices content, marketing materials, running a meeting, helping a user in chat, operating a cluster, responding to a `help wanted` flag, reviewing and commenting on contributions of all these types when in a proposed-state (a _pull request_ or _PR_), giving a talk at a conference, staffing a booth at a conference, and so forth.
+For a longer list, refer to this chapter, "[What Is A Contribution?](https://www.theopensourceway.org/the_open_source_way-guidebook-2.0.html#_what_is_a_contribution)", which includes these definitions:
+
+> An open source **Contribution**
+>
+> Any original, intentional, and substantive object given freely to an open source community, under the licensing of that community. A contribution can come from an individual or a community.
+
+> An open source **Contributor**
+>
+> Any individual person involved in making Contributions to the community. Communities are interpersonal by their nature. They consist of humans, not organizations. Organizations can send their members, staff, leaders, and so forth out to make contributions to a community as a contributor
+
+One reason this project runs so many processes through a git workflow is to raise the visibility on all types of contributions and contributor roles.
+These interactions by a contributor help qualify and quantify their activities.
+It is never a full picture of one's activities, but it is a useful and helpful one.
+
+| Role | Responsibilities | Requirements | Defined by |
```

Comment:
Are these roles defined at the sub-project level or are they Op1st project wide? Are they mutually exclusive, or can I be a member and an approver? Or are they hierarchical, for example, Does the the subproject owner have the same permissions as the approvers and reviewers?   

I think a little more clarity on how these roles function could be helpful here. :)

Reply:
I agree we need to more deeply consider this matrix in comparison with the actual way things are currently laid out and the way we'd like them to evolve. These are intended to be project-wide definitions, as written and sourced (aiui), but that is a difference we can discuss for how we want to approach it.

But I'm not sure that work is a blocker to using this as a starting place, clarifying and fixing in Gov 1.1.

What would be a minimal viable section here?

We can add a sentence about the definitions intended to be project-wide and forthcoming work on the governance will bring more clarity to this section.

If these can be edited to line up better with how we already do project roles, what would those edits look like?

Reply:
Sure, this can be held until Gov 1.1. 

However, just for clarity, currently the use of  OWNERS files is per sub-project (git repo), I'm not sure if there is such a thing as a an org level OWNERS file to define these roles currently. At the org level we have a [github-config](https://github.com/operate-first/common/blob/main/github-config.yaml) where org admins, members and teams can be defined. 

It might make sense to have a note at the beginning like, "Below are the initial roles defined by Governance 1.0, where "Members" are defined across all of Op1st projects, and "Reviews", "Approvers", and "Subproject Owners" are defined per subproject. We will continue to evaluate and refine these role definitions as we move to Governance 1.1"(?)

Reply:
Resolving this to make clear nothing to do here for this PR, but we'll use these resolved conversations to seed the Gov 1.1. discussions.